include/lai/drivers/ec.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Lightweight ACPI Implementation
+ * Copyright (C) 2018-2019 the lai authors
+ */
+
+#pragma once
+
+#include <lai/core.h>
+#include <lai/helpers/resource.h>
+
+#define EC_READ           0x80
+#define EC_WRITE          0x81
+#define EC_BURST_ENABLE   0x82
+#define EC_BURST_DISABLE  0x83
+#define EC_QUERY          0x84
+
+#define EC_STATUS_OBF     0 // Output buffer full
+#define EC_STATUS_IBF     1 // Input buffer full
+#define EC_STATUS_CMD     3 // Internally used
+#define EC_STATUS_BURST   4 // Temporarily halts normal processing so multiple commands can be processed
+#define EC_STATUS_SCI_EVT 5 // SCI event pending
+#define EC_STATUS_SMI_EVT 6 // SMI event pending
```

Comment:
`EC_` constants can move to `include/acpispec` (they are from the spec, right?) and get an `ACPI_EC_` prefix.